MinecraftClient/Commands/Move.cs

```diff
@@ -1,38 +1,36 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using MinecraftClient.Mapping;
 
 namespace MinecraftClient.Commands
 {
     public class Move : Command
     {
         public override string CmdName { get { return "move"; } }
-        public override string CmdUsage { get { return "move <on|off|get|up|down|east|west|north|south|x y z>"; } }
-        public override string CmdDesc { get { return "walk or start walking."; } }
+        public override string CmdUsage { get { return "move <on|off|get|up (-f)|down (-f)|east (-f)|west (-f)|north (-f)|south (-f)|x y z>"; } }
+        public override string CmdDesc { get { return "walk or start walking. Included protection can be bypassed with \"-f\"."; } }
 
         public override string Run(McClient handler, string command, Dictionary<string, object> localVars)
         {
-            string[] args = getArgs(command);
-            string argStr = getArg(command).Trim().ToLower();
+            string[] args = getArgs(command.ToLower());
```

Comment:
I think the parsing of the arguments could be done like below:
https://github.com/MCCTeam/Minecraft-Console-Client/blob/259ef80cf97b18a55da0f4b810460e1e49e1711a/MinecraftClient/Program.cs#L63-L71

We look for `-f` in the whole argument string, if found, set a boolean flag to true and remove the `-f` from the argument string. Then the position of `-f` will no longer be important, and could be easily apply the flag to moving with absolute coordinate.